src/Target.php

```diff
@@ -286,9 +299,153 @@ public function getEnabled(): bool
      */
     protected function getTime($timestamp): string
     {
-        $toString = \str_replace(',', '.', (string) $timestamp);
+        $toString = \str_replace(',', '.', (string)$timestamp);
         $parts = explode('.', $toString);
 
         return date('Y-m-d H:i:s', $parts[0]) . ($this->microtime && isset($parts[1]) ? ('.' . $parts[1]) : '');
     }
+
+    /**
+     * @param $logVars
+     * @return Target
+     */
+    public function setLogVars(array $logVars): self
+    {
+        $this->logVars = $logVars;
+        return $this;
+    }
+
+    /**
+     * @return array
```

Comment:
Such phpdoc is useless - can we remove it if it only duplicates information available in method signature?

Reply:
Yes, we can.

Reply:
Will our apidoc component correctly parse that?